app/controllers/PasskeyController.scala

```diff
@@ -22,10 +26,18 @@ class PasskeyController(
     authAction: AuthAction[AnyContent],
     host: String,
     janusData: JanusData
-)(implicit dynamoDb: DynamoDbClient, mode: Mode, assetsFinder: AssetsFinder)
-    extends AbstractController(controllerComponents)
+)(implicit
+    dynamoDb: DynamoDbClient,
+    mode: Mode,
+    assetsFinder: AssetsFinder,
+    ec: ExecutionContext
+) extends AbstractController(controllerComponents)
     with Logging {
 
+  private def passkeyAuthAction
+      : ActionBuilder[UserIdentityRequest, AnyContent] =
+    authAction.andThen(new PasskeyAuthFilter(host))
```

Comment:
This is implicitly taking the executiuon context being passed to the controller for the construction of the `PasskeyAuthFilter`. Just worth keeping in mind that we're using the application's execution context (threadpool) for the passkey filter as well. Certainly not saying this is wrong, but especially given that we're doing a lot of blocking in the passkey code, it's worth keeping this in mind.

I think we might want to consider this as part of a review and tidy-up once this is all in place. It may be worth using a separate execution context for the passkey code.

Reply:
Just added a todo as a reminder in a28ffa3ab7bd97226948ff4284e6deb34cd403f6